test/index.js

```diff
@@ -15,18 +18,16 @@ const SchwiftyMigration = require('../lib');
 
 const lab = exports.lab = Lab.script({ schedule: false });
 const expect = Code.expect;
-const { describe, it, before } = lab;
+const { describe, it, afterEach } = lab;
 const Utils = require('./utils');
 
 const internals = {};
 
-lab.afterEach((done) => {
-
-    // Just brute clear out the db
+afterEach((done) => {
```

Comment:
Just double-checking that this runs fine for you!  I only ask because it's outside of any `describe()` blocks, which stood out a little bit.